web/templates/web/demographic-data-update.html

```diff
@@ -0,0 +1,54 @@
+{% extends 'web/base.html' %}
+{% load bootstrap3 %}
+{% block title %}Update demographics{% endblock %}
+
+
+{% block flash %}
+  {% bootstrap_messages %}
+  {% if form.non_field_errors %}
+  <div class="alert alert-danger" role="alert">
+      {{ form.non_field_errors }}
+  </div>
+  {% endif %}
+{% endblock %}
+{% block content %}
+    <div class='lookit-row lookit-page-title'>
+        <h2 class='container'>My Account</h2>
+    </div>
+    <div class="container">
+        <div class="row account-edit">
+            <div class="col-md-4 mb-lg">
+                <a class="btn btn-default btn-md btn-block" href="/account"><strong>Account information</strong><br> Change your name, email, or password.</a>
```

Comment:
You could use that partial here.

Alternatively, you could make a "layout" template that this page (and the others like it) could extend, like it currently extends web/base.html and put all your global boilerplate in that. That template could extend web/base.html so you don't have to duplicate it's contents. Then you create your own block in your layout that's inside block content.